functorch/csrc/BatchRulesLinearAlgebra.cpp

```diff
@@ -160,6 +178,7 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   VMAP_SUPPORT("mv", mv_batch_rule);
   VMAP_SUPPORT("mm", mm_batch_rule);
   m.impl("linear", linear_decomp);
+  m.impl("orgqr", orgqr_decomp);
```

Comment:
Without this we get 
`RuntimeError: aten::orgqr hit the vmap fallback which is currently disabled`

But `orgqr` is a composite operator 🤔.

Reply:
Maybe, you can add it into BatchRulesDecompositions.cpp